cvat-ui/src/reducers/index.ts

```diff
@@ -645,6 +658,17 @@ export interface NotificationsState {
             fetching: null | ErrorState;
             canceling: null | ErrorState;
             deleting: null | ErrorState;
+        };
+        selection: {
+            bulkOperation: {
```

Comment:
I would propose to split `selection` logic and `bulkOperation` logic on state level. 
E.g. now in this state nothing is responsible for `selection`, it is only about bulk operation.

Reply:
Selection is only responsible for adding/removing elements from selection IMO. 
Bulk operation is responsible for progress/status/fails.

Reply:
Makes sence, removed.